commcare_connect/opportunity/forms.py

```diff
@@ -822,31 +819,35 @@ def __init__(self, *args, **kwargs):
 
         self.helper = FormHelper(self)
         self.helper.layout = Layout(
-            Row(Field("name")),
-            Row(Field("description")),
-            Row(Field("amount")),
-            Row(Column("start_date"), Column("end_date")),
-            Row(Field("required_deliver_units")),
-            Row(Field("optional_deliver_units")),
-            HTML(
-                f"""
-                <button type="button" class="btn btn-sm btn-outline-info mb-3" id="sync-button"
-                hx-post="{reverse('opportunity:sync_deliver_units', args=(org_slug, opportunity_id))}"
-                hx-trigger="click" hx-swap="none" hx-on::after-request="alert(event?.detail?.xhr?.response);
-                event.detail.successful && location.reload();
-                this.removeAttribute('disabled'); this.innerHTML='Sync Deliver Units';""
-                hx-disabled-elt="this"
-                hx-on:click="this.innerHTML=&quot;<span class=\\
-                'spinner-border spinner-border-sm'></span> Syncing...&quot;;">
-                <span id="sync-text">Sync Deliver Units</span>
-                </button>
+            Div(
+                Row(
+                    Column(Field("name"), Field("description")),
```

Comment:
This `Column` class can be removed as this will only add a wrapping div with no style classes assigned to it.